lib/katello/plugin.rb

```diff
@@ -302,6 +303,370 @@
+        type: :string,
+        default: 'Kickstart default PXELinux',
+        full_name: N_('Default synced OS PXELinux template'),
+        description: N_("Default PXELinux template for new Operating Systems created from synced content"),
+        collection: proc { katello_template_setting_values("PXELinux") }
+
+      setting 'katello_default_PXEGrub',
+        type: :string,
+        default: 'Kickstart default PXEGrub',
+        full_name: N_('Default synced OS PXEGrub template'),
+        description: N_("Default PXEGrub template for new Operating Systems created from synced content"),
+        collection: proc { katello_template_setting_values("PXEGrub") }
+
+      setting 'katello_default_PXEGrub2',
+        type: :string,
+        default: 'Kickstart default PXEGrub2',
+        full_name: N_('Default synced OS PXEGrub2 template'),
+        description: N_("Default PXEGrub2 template for new Operating Systems created from synced content"),
+        collection: proc { katello_template_setting_values("PXEGrub2") }
+
+      setting 'katello_default_iPXE',
+        type: :string,
+        default: 'Kickstart default iPXE',
+        full_name: N_('Default synced OS iPXE template'),
+        description: N_("Default iPXE template for new Operating Systems created from synced content"),
+        collection: proc { katello_template_setting_values("iPXE") }
+
+      setting 'katello_default_ptable',
+        type: :string,
+        default: 'Kickstart default',
+        full_name: N_('Default synced OS partition table'),
+        description: N_("Default partitioning table for new Operating Systems created from synced content"),
+        collection: proc { Hash[Template.all.where(:type => "Ptable").map { |tmp| [tmp[:name], tmp[:name]] }] }
+
+      setting 'katello_default_kexec',
+        type: :string,
+        default: 'Discovery Red Hat kexec',
+        full_name: N_('Default synced OS kexec template'),
+        description: N_("Default kexec template for new Operating Systems created from synced content"),
+        collection: proc { katello_template_setting_values("kexec") }
+
+      setting 'katello_default_atomic_provision',
+        type: :string,
+        default: 'Atomic Kickstart default',
+        full_name: N_('Default synced OS Atomic template'),
+        description: N_("Default provisioning template for new Atomic Operating Systems created from synced content"),
+        collection: proc { katello_template_setting_values("provision") }
+
+      setting 'manifest_refresh_timeout',
+        type: :integer,
+        default: 60 * 20,
+        full_name: N_('Timeout when refreshing a manifest (in seconds)'),
+        description: N_("Manifest refresh timeout")
+
+      setting 'content_action_accept_timeout',
+        type: :integer,
+        default: 20,
+        full_name: N_('Accept action timeout'),
+        description: N_("Time in seconds to wait for a host to pick up a katello-agent action")
```

Comment:
changed these descriptions to make it more clear that they apply only for katello-agent and not REX